tests/source/issue-5496/70.rs

```diff
@@ -0,0 +1,8 @@
+// rustfmt-version: One
+// rustfmt-single_line_if_else_max_width: 70
+fn foo() -> usize {
+    let some_long_name = true;
+    let some_other_long_name = false;
+    let bar = if some_long_name && some_other_long_name { baz() } else { buzz() };
+    if some_long_name && some_other_long_name { 1 } else { 2 }
+}
```

Comment:
Not a big deal, but would be nice to have a newline at the end of the file. I believe most `source` tests are missing the trailing newline.

Reply:
Thanks, done!